addon.gypi

```diff
@@ -109,21 +109,22 @@
         ],
       }],
       [ 'OS=="zos"', {
-        'cflags': [
-          '-q64',
-          '-Wc,DLL',
-          '-qlonglong',
-          '-qenum=int',
-          '-qxclang=-fexec-charset=ISO8859-1'
+        'conditions': [
+          [ '"<!(echo $CC)" != "clang" and "<!(echo $CC)" != "ibm-clang64"', {
+            'cflags': [
+              '-q64',
+              '-Wc,DLL',
+              '-qlonglong',
+              '-qenum=int',
+              '-qxclang=-fexec-charset=ISO8859-1'
+            ],
+          }],
+        ],
+        'ldflags': [
+          '<(node_exp_file)',
         ],
         'defines': [
-          '_ALL_SOURCE=1',
```

Comment:
Should we leave these defines in, in case the common.gypi is not included (as in earlier editions of v14)?

Reply:
Ok, but will remove the `=1` to match those in common.gypi and also how they're used (`#ifdef ...` instead of `#if ...`).